app/views/recipes/toggle_public.js.erb

```diff
@@ -0,0 +1,3 @@
+// Update the checkbox state
+var isChecked = <%= @recipe.public ? 'true' : 'false' %>;
+$('#public-toggle').prop('checked', isChecked);
```

Comment:
- [ ] This radio box displayed on the `recipe_show_page` is not functional logically (I was only able to toggle it ), kindly address it. ℹ️ 

<img width="584" alt="Untitled" src="https://github.com/agneta1991/recipe_app/assets/103386183/836cd486-9c8b-4bd0-b3ea-19e55f93f9e6">

Reply:
@OyePriscilla this piece of code was left there by accident, after removing it, the button works perfectly and it updates the recipe public to private or other way round as expected, it has been checked before and after toggling it in the database.